src/sql/workbench/common/sqlWorkbenchUtils.ts

```diff
@@ -19,3 +19,41 @@ export function getSqlConfigValue<T>(workspaceConfigService: IConfigurationServi
 	let config = workspaceConfigService.getValue<{ [key: string]: any }>(ConnectionConstants.sqlConfigSectionName);
 	return config ? config[configName] : undefined;
 }
+
+/**
+ * Wraps provided string using Unicode character U+000A LINE FEED (LF) that qualifies as line break to wrap text in title attributes (tooltips).
+ * @param str string to be wrapped
+ * @param maxWidth max width to be allowed for wrapped text
+ * @returns wrapped string
+ */
+export function wrapStringWithUnicodeNewLine(str: string, maxWidth: number): string {
+	if (!str) {
+		return null;
```

Comment:
1. The parameter should be `str: string | undefined` if it's optional
2. You should return undefined, not null (convention we use in ADS)
3. The return type should be `string | undefined` (after the change in 2)

But in general I don't know if we need to be this defensive. We're the only ones calling this code and anyone calling it should be able to make sure that it isn't passed an undefined value. It's not a huge deal to add the extra check but right now it'd never even possibly get used (because you already check title before passing it in here)